.gitpod.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM gitpod/workspace-mysql
+
+USER root
+
+# Update APT Database
+### base ###
+RUN sudo apt-get update -q \
+    && sudo apt-get install -y php-dev
+
+# Install XDebug
+RUN wget http://xdebug.org/files/xdebug-2.9.1.tgz \
+    && tar -xvzf xdebug-2.9.1.tgz \
```

Comment:
You can combine the download + extraction with:

```
curl -sSf http://xdebug.org/files/xdebug-2.9.1.tgz | tar xz
```

(so that we only have the `xdebug-2.9.1` directory without a useless `xdebug-2.9.1` file)

Reply:
thanks for your feedback